assets/js/modules/pagespeed-insights/components/common/FieldReportMetrics.js

```diff
@@ -118,9 +117,6 @@ export default function FieldReportMetrics( { data, error } ) {
 					/>
 				</tbody>
 			</table>
-			<div className="googlesitekit-pagespeed-report__row googlesitekit-pagespeed-report__row--single-item googlesitekit-pagespeed-report__row--last">
```

Comment:
The `googlesitekit-pagespeed-report__row--last` class needs to move up to the row for "Cumulative Layout Shift" as it is on the Lab Data tab to prevent a layout shift when toggling between tabs 😄 

See here https://google.github.io/site-kit-wp/storybook/pull/2480/?path=/story/pagespeed-insights-module-components--dashboard-widget

Also, the lighthouse recommendations are missing in Storybook, but not sure if that's expected or a blocker here.